ml4gw/waveforms/ringdown.py

```diff
@@ -54,37 +58,43 @@ def __call__(
         # add dimension for calculating waveforms in batch
         frequency = frequency.view(-1, 1)
         quality = quality.view(-1, 1)
-        amplitude = amplitude.view(-1, 1)
+        epsilon = epsilon.view(-1, 1)
         phase = phase.view(-1, 1)
         inclination = inclination.view(-1, 1)
-        #distance = distance.view(-1, 1)
+        distance = distance.view(-1, 1)
+
+        distance = distance*(3.086e22) #conveting to meters from MPC
 
         # ensure all inputs are on the same device
         pi = torch.tensor([torch.pi], device=frequency.device)
 
+        # Calculate spin 
+        spin = 1 - (2 / quality)**(20 / 9)
+        # Calculate mass 
+        mass = (1 / (2 * pi)) * (c**3 / (G * frequency)) * (1 - 0.63 * (2 / quality)**(2 / 3))
+
+        # Calculate amplitude 
+        F_Q = 1 + (1.34 / quality**2)
+        g_a = (1 + 0.1 * spin**2)
+        amplitude = np.sqrt(5 / 2) * epsilon * (6.67430e-11 * mass / (3e8**2)) * quality**(-0.5) * F_Q**(-0.5) * g_a**(-0.5)
```

Comment:
Need to switch to using `c` and `G` here

Reply:
Also, should have mentioned this earlier: we use a tool called `pre-commit` to automatically keep code tidy. In the top-level of the repo, can you run `poetry run pre-commit install`? Then, each time you make a `git commit`, it will check over your code, halt the commit if it finds any problems, and try to reformat it.